package.json

```diff
@@ -30,6 +31,12 @@
     },
     "author": "Nordic Semiconductor ASA",
     "license": "Proprietary",
+    "binary": {
+        "module_name": "pc-nrfjprog-js",
+        "module_path": "./build/Release",
+        "host": "https://github.com/NordicSemiconductor/pc-nrfjprog-js/releases/download/",
+        "remote_path": "v{version}"
+    },
```

Comment:
Why was this copied here?

Reply:
For internal release usage. The launcher needs this code here to download prebuild binaries in the draft release of pc-nrfjprog-js.